### PR TITLE
fix(i18n/error): more helpful trip plan failure message

### PR DESCRIPTION
**Pull request by miles-grant-ibigroup**
This PR makes a small adjustment to the trip not found error message to provide a helpful pointer to the trip settings form.

### CODE REVIEW

File: lib/util/state.js [hunk 1]
```diff
@@ -163,7 +164,7 @@ export function getErrorMessage(error, intl) {
         id="util.state.noTripFoundWithReason"
         values={{
           noTripFound: noTripFoundText,
-          reason: <FormattedMessage id="util.state.noTripFoundReason" />
+          reason: <FormattedMessage id="util.state.noTripFoundReason" values={{strong:Strong}}/>,
```
Comment (binh-dam-ibigroup) [inline lib/util/state.js:167]: This wasn't prettified?

File: i18n/en-US.yml [hunk 2]
```diff
@@ -857,7 +857,7 @@ util:
     networkUnavailable: The {network} network is unavailable at this moment.
     noTripFound: No trip found.
     noTripFoundForMode: No trip found for {modes}.
-    noTripFoundReason: There may be no transit service within the maximum specified distance or at the specified time, or your start or end point might not be safely accessible.
+    noTripFoundReason: There may be no transit service within the maximum specified distance or at the specified time, or your start or end point might not be safely accessible. <strong>An itinerary may be found if you adjust some Transit Options & Preferences above.</strong>
```
Comment (binh-dam-ibigroup) [inline i18n/en-US.yml:860]: Place the <strong> tag in a separate line.

Comment (binh-dam-ibigroup) [reply i18n/en-US.yml:860]: How about replacing "An itinerary may be found..." with "Try adjusting your search options"? (Careful, in batch routing, there is no "Transit Options & Preferences" text, and there is no "above" in mobile (you have to click Edit to change the search).

**Comment by binh-dam-ibigroup** [comment]
@miles-grant-ibigroup Let me know how you would like to handle https://github.com/opentripplanner/otp-react-redux/pull/535/files#r794546688.

File: i18n/en-US.yml (hunk 2 above)
Comment (miles-grant-ibigroup) [reply i18n/en-US.yml:860]: How about "Try editing your search preferences" that covers all cases -- preferences hints towards the non-batch mode, edit hints towards the mobile edit button, and search preferences hints towards the batch mode gear

Comment (binh-dam-ibigroup) [reply i18n/en-US.yml:860]: That works for me.

Comment (miles-grant-ibigroup) [reply i18n/en-US.yml:860]: Yes the strong text was requested by users

File: lib/util/state.js (hunk 1 above)
Comment (miles-grant-ibigroup) [reply lib/util/state.js]: Not sure what happened there! Repaired in a26a1d74f96c8a675361ddf0b58bb0e4b3c9323a

File: i18n/en-US.yml (hunk 2 above)
Comment (miles-grant-ibigroup) [reply i18n/en-US.yml]: Removed the hardcoded string update in a26a1d74f96c8a675361ddf0b58bb0e4b3c9323a

**Review by binh-dam-ibigroup — APPROVED** [review]

**Comment by daniel-heppner-ibigroup** [comment]
What exactly is going on here? I see that you've added Strong to the formatted text, but then removed the string that would actually use that.

**Comment by miles-grant-ibigroup** [comment]
The `<strong>` is there to allow strings added via `config.yml` to use `<strong>`

**Review by daniel-heppner-ibigroup — APPROVED** [review]

File: lib/util/state.js [hunk 3]
```diff
@@ -163,7 +164,12 @@ export function getErrorMessage(error, intl) {
         id="util.state.noTripFoundWithReason"
         values={{
           noTripFound: noTripFoundText,
-          reason: <FormattedMessage id="util.state.noTripFoundReason" />
+          reason: (
+            <FormattedMessage
+              id="util.state.noTripFoundReason"
+              values={{ strong: Strong }}
```
[the remark this answers is not in the document]
Comment (miles-grant-ibigroup) [reply lib/util/state.js:170]: This allows strong to be used in the override.

**Pull request merged**